internal/pkg/api/error.go

```diff
@@ -33,22 +33,24 @@ const (
 	LogAccessAPIKeyID = logger.AccessApiKeyId
 )
 
-type errResp struct {
+// ErrResp is an HTTP error response
+type ErrResp struct {
```

Comment:
[Suggestion]
If it's specific for HPPT, wouldn't be better to call it `ErrHTTPResp`?